blog/2023-06-23-newsletter.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "Conda News, Issue #1!"
+slug: "2023-06-23-newsletter"
+description: |
+    Recent and upcoming activity in the conda community.
+authors: [tnabtaf, pinak-datta]
+tags: [conda, newsletter]
+image: img/blog/2023-06-23-newsletter/balloon-launch.jpg
+---
+
+import Image from '@theme/IdealImage';
+import launch from '@site/static/img/blog/2023-06-23-newsletter/balloon_launch.jpg';
+import scipybanner from '@site/static/img/blog/2023-06-23-newsletter/scipy_banner.png';
+import packagingcon from '@site/static/img/blog/2023-06-23-newsletter/packagingcon.jpg';
+import rattler from '@site/static/img/rattler.png';
+import youtube from '@site/static/img/blog/2023-06-23-newsletter/youtube.png';
+import cf from '@site/static/img/conda_forge_wide.png';
+import bioconda from '@site/static/img/bioconda_word_logo.png';
+import forum from '@site/static/img/blog/2023-06-23-newsletter/conda_community_forum.png';
+
+<Image img={launch} alt="Banner image for blog post; The Conda Newsletter is launching - hot air balloon inflating"/>
+
+<p>
+    <i>
+        <small>
+            Image credit: <a href="https://unsplash.com/photos/sN40hKv8_cI">Road Trip with Raj on Unsplash</a>
+        </small>
+    </i>
+</p>
+
+**Welcome to the first ever conda community newsletter!** The newsletter will highlight recent and upcoming activity that is relevant to the [conda community](https://conda.org/). This first issue highlights releases, grants, events, ....
```

Comment:
We have something else we want to add to this list?

Reply:
Ah, I am guessing that you don't like that ellipsis at the end.

I will make this list exhaustive and explicit.